specification/containerinstance/resource-manager/Microsoft.ContainerInstance/stable/2018-10-01/examples/ContainerGroupsCreateOrUpdate.json

```diff
@@ -0,0 +1,275 @@
+{
+  "parameters": {
+    "subscriptionId": "subid",
+    "api-version": "2018-10-01",
+    "resourceGroupName": "demo",
+    "containerGroupName": "demo1",
+    "containerGroup": {
+      "id": "/subscriptions/subid/resourceGroups/demo/providers/Microsoft.ContainerInstance/containerGroups/demo1",
+      "location": "west us",
+      "name": "demo1",
+      "identity": { 
+        "type": "SystemAssigned, UserAssigned",
+        "UserAssignedIdentities": {
```

Comment:
Casing for this one is also wrong, see [log](https://travis-ci.org/Azure/azure-rest-api-specs/jobs/436305779).  I guess the tool only reports one error at a time per example.  :(  You can run it locally, see instructions [here](https://github.com/Azure/oav).